keyboards/id80/ansi/keymaps/msf/config.h

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
done. thanks